### PR TITLE
Improve transport documentation

### LINKED ISSUE #1501: Add missing end-user transport documentation
- `containers-transports.5.md` is missing at least `sif:`
- `containers-policy.json.5.md` is missing quite a few transports

… so add that, and figure out some safeguards to decrease the risk of this happening again.

### PR DESCRIPTION
**Pull request by mtrmac**
… in `containers-transports(5)` and `containers-policy.json(5)`.

See individual commit messages for a bit more detail.

Fixes #1501 .

### CODE REVIEW

**Review by vrothberg — APPROVED** [review]
LGTM

File: docs/containers-policy.json.5.md [hunk 1]
```diff
@@ -61,29 +61,52 @@ The global `default` set of policy requirements is mandatory; all of the other f
...
+
+Supported scopes have the form `[`_storage-specifier_`]`_image-scope_.
+
+`[`_storage-specifier_`]` is usually `[`_graph-driver-name_`@`_graph-root_`]`, e.g. `[overlay@/var/lib/containers/storage]`.
+
+_image-scope_ matching the individual image is
+- a named Docker reference *in the fully expanded form*, either using a tag or digest. For example, `docker.io/library/busybox:latest` (*not* `busybox:latest`)
+- and/or (depending on which one the user’s input provides) `@`_image-id_
+
+More general scopes are prefixes of individual-image scopes, and specify a less-precisly-specified image, or a repository
```
Comment (TomSweeneyRedHat) [inline docs/containers-policy.json.5.md:88]:
```suggestion
More general scopes are prefixes of individual-image scopes, and specify a less-precisely-specified image, or a repository
```

**Comment by TomSweeneyRedHat** [comment]
One small typo, otherwise LGTM.

File: docs/containers-policy.json.5.md (hunk 1 above)
Comment (mtrmac) [reply docs/containers-policy.json.5.md]: Thanks! Fixed.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**